src/components/calcite-sortable-list/calcite-sortable-list.tsx

```diff
@@ -44,6 +46,11 @@ export class CalciteSortableList {
    */
   @Prop() handleSelector = "calcite-handle";
 
+  /**
+   * Indicates the horizontal or vertical orientation of the component.
+   */
+  @Prop({ reflect: true }) layout: Layout = "vertical";
```

Comment:
Does it need to be reflected? Seems like it doesn't.

Reply:
Makes sense. Will update.